src/pages/extending-chat-widget/customer-sdk/v2.0/index.mdx

```diff
@@ -95,15 +95,16 @@ You will need to provide your licenseId and clientId when creating a Customer SD
-| clientId             | string             |         | Your client id, you receive this value when you register your application in the LiveChat developer console.                                                                  |
-| autoConnect          | boolean            | true    | Optional; should the library try to reconnect on it's own.                                                                                                                    |
-| groupId              | number             | 0       | Optional; the id of the group you wish to connect to.                                                                                                                         |
-| region               | 'dal' \| 'fra'     | 'dal'   | Optional; the server region your license is at.                                                                                                                               |
-| redirectUri          | string             |         | Optional; should only be used inside ReactNative, this is the URI which your webview is redirected to after authorization.                                                    |
-| customerDataProvider | () => CustomerData |         | Optional; should only be used if you need to send customer data during login. In general, [`updateCustomer()`](#updateCustomer) should be prefered for sending customer data. |
+| parameters           | type               | default | description                                                                                                                                                                                                                     |
+| -------------------- | ------------------ | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| licenseId            | number             |         | Your license number, you receive this value when creating a new livechat account.                                                                                                                                               |
+| clientId             | string             |         | Your client id, you receive this value when you register your application in the LiveChat Developer Console.                                                                                                                    |
+| autoConnect          | boolean            | true    | Optional; should the library try to reconnect on it's own.                                                                                                                                                                      |
```

Comment:
```suggestion
| autoConnect          | boolean            | true    | Optional; should the library try to reconnect on its own.                                                                                                                                                                      |
```

Reply:
Not sure I understand the "should the library" construction 🤔

Reply:
I'll fix that in both versions, but what would you suggest? That parameter decides whether or not the SDK should reconnect on its own, so WDYT about:

```suggestion
| autoConnect          | boolean            | true    | Optional; decides whether or not the SDK should reconnect on its own |
```

Reply:
Ok! The new version's fine. We can skip "or not" because whether already means that. And add a full stop at the end :)